bindings/java/src/main/native/org_ucx_jucx_ucp_UcpContext.h

```diff
@@ -0,0 +1,29 @@
+/* DO NOT EDIT THIS FILE - it is machine generated */
```

Comment:
can we generate this file during build process, instead of saving it in source control?